### PR TITLE
edited readme and install-dev.sh

### PR DESCRIPTION
**Pull request by thebigbone**

### CODE REVIEW

File: install-dev.sh [hunk 1]
```diff
@@ -1,4 +1,15 @@
 #!/bin/sh
+
+# Check if pip is installed, and install it if necessary
+if ! command -v pip >/dev/null 2>&1; then
+    echo "---- Installing pip ----"
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```
Comment (c3b5aw) [inline install-dev.sh:6]: Is that possible to automatically remove the pip install file ?

File: install-dev.sh [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    python get-pip.py
+else
+    echo "---- Pip is already installed ----"
+fi
+
+
 echo "---- Installing Python Poetry ----"
 pip install -U pip
```
Comment (c3b5aw) [inline install-dev.sh:14]: You could move that line up and say "updating pip"

Comment (thebigbone) [reply install-dev.sh:14]: Do you want me to remove the check which was done by the if statement?

File: install-dev.sh (hunk 1 above)
Comment (thebigbone) [reply install-dev.sh:6]: Yeah. I could add rm get-pip.py after that.

File: install-dev.sh [hunk 3]
```diff
@@ -1,4 +1,12 @@
 #!/bin/sh
+
+# Check if pip is installed, and install it if necessary
+echo "---- Updating pip ----"
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+echo "---- Pip updated ----"
+rm get-pip.py
+
```
Comment (c3b5aw) [inline install-dev.sh:9]:
```suggestion
if ! pip --version > /dev/null 2>&1; then
    echo "Installing pip..."
    curl -sSL https://bootstrap.pypa.io/get-pip.py | python
fi
```

File: install-dev.sh [hunk 4]
```diff
@@ -1,4 +1,12 @@
...
 echo "---- Installing Python Poetry ----"
 pip install -U pip
```
Comment (c3b5aw) [inline install-dev.sh:11]:
```suggestion
echo "---- Updating pip ----"
pip install -U pip
```

**Review by c3b5aw — CHANGES_REQUESTED** [review]

File: install-dev.sh [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
+echo "---- Updating pip ----"
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+echo "---- Pip updated ----"
+rm get-pip.py
+if ! pip --version > /dev/null 2>&1; then
+    echo "Installing pip..."
+    curl -sSL https://bootstrap.pypa.io/get-pip.py | python
+fi
 echo "---- Installing Python Poetry ----"
```
Comment (c3b5aw) [inline install-dev.sh:13]: Would have put that line down ahah

**Review by c3b5aw — APPROVED** [review]

File: install-dev.sh (hunk 5 above)
Comment (thebigbone) [reply install-dev.sh:13]: Are you going to merge it?

Comment (c3b5aw) [reply install-dev.sh:13]: Yes ! We have to resolve the comments though ☺️

**Review by c3b5aw — APPROVED** [review]

**Comment by nullswan** [comment]
Merged, thanks for the MR @thebigbone!

**Comment by thebigbone** [comment]
> Oh, you needed your commit to be signed.
> 
> image
> 
> Make sure you work on having a gpg key to sign your commit if you contribute more in the open-source world.

Oops, sorry. I will definitely set up a gpg key for future commits. Thanks for the advice!

**Pull request merged**